### DISCUSSION TITLE
Not a code issue but had no idea where to post

### DISCUSSION THREAD

**Discussion by seanaguinaga** [FirebaseExtended/reactfire#222] [General]
Hey guys!

Thank you SO much for working on reactifre and congrats on 2.0!!

I've been on canary for a while and I had no idea it was released until I ran ncu -u on my project!

is there a website or mailing list or anything of that ilk for reactfire news, updates, feature showcases, etc? Medium posts? etc? 

it's such a cool library!

Thanks
Sean

**Comment by jhuleatt** [comment]
Hey Sean, awesome to hear you're excited about the library 😃 

We're still early days so don't have much besides my [initial talk](https://youtu.be/Mi9aKDcpRYA?t=527), but we're working on more developer content. James, David, and I will tweet about any new content that comes out, so that's probably the most reliable place to look for now (Twitter handles: [jhuleatt](https://twitter.com/jhuleatt), [_davideast](https://twitter.com/_davideast), [jamesuriah](https://twitter.com/jamesuriah)).

**Comment by seanaguinaga** [comment]
I will follow you guys!

in case you were thinking of spectrum for a community page - tanner linsley from react static pointed out their pages don't come up in google search results. Might be something to keep in mind! 

I know it doesn't look it but I'm a decent writer if you need any contributions.

**Comment by seanaguinaga** [comment]
Also, unrelated, would you ever mix reactfire with mobx?

**Comment by jhuleatt** [comment]
> I'm a decent writer if you need any contributions.

Great! We're always open to PRs with new use cases for the [guide docs](https://github.com/FirebaseExtended/reactfire/blob/master/docs/use.md#using-reactfire). A couple of opportunities off the top of my head:

- Listen to a firestore query and render the results (`useFirestoreCollectionData`)
- Listen to a Remote Config value

If there are any common use cases you'd like to document but you can't figure out how to implement, feel free to open an issue to discuss :)

> would you ever mix reactfire with mobx?

Honestly, I've never used mobx, so I'm not sure what an integration would look like. If we need to change the library in some way to support mobx, please open an issue and we can talk through it.
